Qpf/Macro/Data/View.lean

```diff
@@ -122,11 +122,15 @@ def DataView.addDeclNameSuffix (view : DataView) (suffix : String) : DataView
 
 
 /-- Returns the fully applied form of the type to be defined -/
-def DataView.getExpectedType (view : DataView) : Term
-  := Syntax.mkApp (mkIdent view.shortDeclName) (
+def DataView.getExpectedTypeWithId (view : DataView) (id : Ident) : Term
```

Comment:
Please explain in the docstring what `id` does, also, here, too I feel `Name` is more appropriate
```suggestion
def DataView.getExpectedTypeWithId (view : DataView) (id : Name) : Term
```